packages/sampler/stories/qa/Header.js

```diff
@@ -102,9 +102,10 @@ storiesOf('Header', module)
 		'with long text and headerComponent',
 		() => (
 			<Header
-				title={text('title', Config, 'Title')}
+				title={text('title', Config, 'Title is very very very very very very very long.')}
 				titleBelow={text('titleBelow', Config, 'This is a header sample with long titleBelow text and header components to test positioning of header components.')}
 				subTitleBelow={text('subTitleBelow', Config, 'This is a header sample with long titleBelow, subTitleBelow text and header components to test positioning of header components.')}
+				marqueeOn={select('marqueeOn', ['', 'hover', 'render'], Header, '')}
```

Comment:
Please remove empty and use with `default` value

Reply:
@seunghoh 
I removed the empty default value.